### ISSUE TITLE
Pub issue

### ISSUE THREAD

**Issue by rightisleft** [eredo/dartson#29]
I'm seeing an odd issue with Dartson & mongo_dart.

I was running mongo_dart 0.1.46. - I need to upgrade it to mongo_dart 0.2.4

However, when i upgrade mongo_dart - i get the following error about dartson

``` 
Package mongo_dart has no versions that match 0.2.4 derived from:
- tickets depends on version 0.2.4
```
using: https://gist.github.com/rightisleft/ab9f49d2a6197658ca06
This is odd, as mongo_dart: '0.2.4' exists on pub.dartlang, and it works in isolation

----

By testing the removal of each individual package in pubspec.yaml, i was able to identify a conflict with dartson.

If i move dartson to the top of the dependency list i get the reverse:
https://gist.github.com/rightisleft/bd830c18df158921d6ea
```
Package dartson has no versions that match 0.2.4 derived from:
- tickets depends on version 0.2.4
```
Removing dartson all together allows for mongo_dart to install correctly.

Is there a known issue between the two libraries? Is there a conflict thats not getting surfaced?

**Comment by rightisleft** [comment]
Running with verbose shows the dependency mismatch.

PR here: https://github.com/eredo/dartson/pull/30

**Comment by eredo** [comment]
Thanks for the PR. Dependency issue resolved in version 0.2.5
